components/Map/index.jsx

```diff
@@ -26,6 +31,40 @@ const Tile = ({ url, name, checked }) => (
   </LayersControl.BaseLayer>
 );
 
+const filters = [
+  {
+    name: 'Sticker',
+    type: 'sticker',
+    checked: true
+  },
+  {
+    name: 'Picture',
+    type: 'picture',
+    checked: true
+  },
+  {
+    name: 'Special',
+    type: 'special',
+    checked: true
+  }
+];
+
+function filterPins(name, type, checked, pins) {
+  var list = pins.filter((pin) => {
```

Comment:
```suggestion
  const list = pins.filter((pin) => {
```

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const).

Reply:
Got it, so here since we won't be changing the value of `list` we should declare it as a constant. Thanks :)